docs/getting_started.md

```diff
@@ -11,11 +11,24 @@ weight=1
 
 # Getting started with Docker Notary
 
-This document describes basic use of Notary as a tool supporting content trust.
-For more advanced use cases, you must [run your own Notary
+This document describes basic use of the Notary CLI as a tool supporting Docker
+Content Trust. For more advanced use cases, you must [run your own Notary
 service](running_a_service.md) and should read the [use the Notary client for
 advanced users](advanced_usage.md) documentation.
 
+## What is Notary
+
+Notary is a tool for publishing and managing trusted collections of content.
+Publishers can digitally sign collections and consumers can verify integrity
+and origin of content. This ability is built on a straightforward key management
+and signing interface to create signed collections and configure trusted publishers.
+
+With Notary anyone can provide trust over arbitrary collections of data. Using
+[The Update Framework (TUF)](http://theupdateframework.com/) as the underlying
```

Comment:
Is this their canonical URL? It redirects to https://theupdateframework.github.io. If it's the canonical URL, looks like you can change it to `https://....` :innocent: